gini/gini-vision-lib-ios
# Localizes image assets

### Description
Added localized image assets for open with tutorial

### How to test
Edit the _Example App_ scheme in Xcode and select either English or German as the app language. Then open the open with tutorial and check that the images are in the correct language.

### Merging
Automatic
